config/ompi_setup_prrte.m4

```diff
@@ -95,6 +95,9 @@ AC_DEFUN([OMPI_SETUP_PRRTE],[
     AM_CONDITIONAL([OMPI_WANT_PRRTE],
            [test "$prrte_setup_internal_happy" = "1" -o "$prrte_setup_external_happy" = "1"])
 
+    AM_CONDITIONAL([OMPI_EXTERNAL_PRRTE],
```

Comment:
Why do this rather than AC_DEFINEing the data you actually want?  The only time to use -D as CPPFLAGS is when you expect that variable to change between configure and build time (like some of the prefix options).

Reply:
I did it this way to limit the scope to just mpirun. I can make the change though to AC_DEFINE it.

Reply:
Traditionally, we haven't worried about that scoping.  And the AM_CONDITIONAL + -D bites us in the butt just often enough to make it work avoiding.